static-assets/components/cstudio-admin/mods/content-types.js

```diff
@@ -85,11 +85,6 @@
         $('#createNewContentTypeButton').click(() => {
           this.onNewClick();
         });
-
-        CStudioAuthoring.ContextualNav.AdminConsoleNav.initActions([
```

Comment:
Now that the contextual-nav.xml configuration is not loaded anymore, `AdminConsoleNav` module is not loaded, so this initialization is not needed (and the object doesn't exist). Checked for other usages and they're dead code (we should remove it)

Reply:
go ahead